cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaMirrorMakerAssemblyOperator.java

```diff
@@ -123,4 +135,33 @@ public KafkaMirrorMakerAssemblyOperator(Vertx vertx, boolean isOpenShift,
         
         return result;
     }
+
+    Future mirrorMakerInitServiceAccount(String namespace, KafkaMirrorMakerCluster mirror) {
+        return serviceAccountOperations.reconcile(namespace,
+                KafkaMirrorMakerCluster.initContainerServiceAccountName(mirror.getCluster()),
+                mirror.generateInitContainerServiceAccount());
+    }
+
+    Future mirrorMakerInitClusterRoleBinding(String namespace, String name, KafkaMirrorMakerCluster mirror) {
+
+        KubernetesClusterRoleBinding desired = mirror.generateClusterRoleBinding(namespace);
+        Future<ReconcileResult<KubernetesClusterRoleBinding>> fut = clusterRoleBindingOperations.reconcile(
+                KafkaMirrorMakerCluster.initContainerClusterRoleBindingName(namespace, name), desired);
+
+        Future replacementFut = Future.future();
+
+        fut.setHandler(res -> {
+            if (res.failed()) {
+                if (desired == null) {
+                    replacementFut.complete();
+                } else {
+                    replacementFut.fail(res.cause());
+                }
+            } else {
+                replacementFut.complete();
+            }
+        });
+
+        return replacementFut;
```

Comment:
This whole function seems to be repeated three times in this PR. Is there some way you can factor it out into a common method/function somewhere?